dotnet/agent-nr-config/Dockerfile

```diff
@@ -23,14 +23,12 @@ COPY --from=build /app/build .
 COPY --from=build /tracer-home /tracer-home
 # The most important environment variables are shown in this example.
 # For a complete list of options refer to the OpenTelemetry documenations available at
-# https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/47f16b5748218f37dd9bd543a0f133670904f9f7/docs/config.md.
+# https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/config.md.
```

Comment:
You might want to pin the documentation link to the v1.0.0-rc.2 tag so that documentation will match agent version being used.